pgml-docs/docs/gym/announcement.md

```diff
@@ -0,0 +1,70 @@
+---
+hide:
+  - navigation
+  # - toc
+  - comments
+---
+
+# Announcing the PostgresML Gym!
+
+Today we're thrilled to announce the launch of our brand new offering: a **free tier** of hosted PostgresML for machine learning enthusiasts and hobbyists! We affectionatly call it "the gym" as its goal is to train one's ML muscle with practical tutorials touching common topics such as generic regression, classification and fraud prediction.
+
+Sign up takes just a few seconds and you'll get access to your own personal PostgresML instance.
+
+<br />
+
+<p align="center" markdown>
+  [Sign Up](https://gym.postgresml.org/signup/){ .md-button .md-button--primary .md-button }
+</p>
+
+
+## Notebooks for Postgres
+
+One big piece of novelty we think is exciting is our freshly designed Notebooks. They work much like the Jupyter Notebooks we love and use with Python, except ours are tailored towards building production-ready models with SQL.
+
+
+
+<div class="grid bare" markdown>
+  <div class="card" markdown>
+![Notebook](./notebook_2.png)
+  </div>
+  <div class="card" markdown>
+
+<br />
+
+Notebooks add a lot of dynamism to what is usually a world of static CSV files.
+
+With SQL, you'll be able to query and filter live datasets, train dozens of different models, and compare results in just a few commands.
+
+All results are saved in Notebook cells, so you can analyze them and share with your friends and colleagues.
+
+<br />
+
+<p align="center" markdown>
+:material-arrow-right: Try this yourself [at the gym!](https://gym.postgresml.org/signup/)
+<p>
+</div>
+</div>
+
+## Tutorials
+
+You can use PostgresML for pretty much any experiments that use relational data (or any data really), and to get you started we've added five (5) tutorials: regression, classification, fraud prediction.
+
+<div class="grid bare" markdown>
+<div class="card" markdown>
+Regression is one of the most common topics in Machine Leanrning.	
+</div>
+<div class="card" markdown>
+**image**
+</div>
+</div>
+
+
+## Open Source
+
+PostgresML Notebooks, PostgresML itself, and our tutorials are open source and are available in our Github [repository](https://github.com/postgresml/postgresml). If you don't feel like joining our gym, nothing personal, you can run the entire experience from your laptop using our Docker deployment.
+
+
+## About PostresML
```

Comment:
I think this needs to be part of the primary value prop explanation. I think no one will remember what PGML is exactly.